tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/StorageAnimalList.java

```diff
@@ -0,0 +1,66 @@
+package com.sci.bv.tema14Ianuarie;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class StorageAnimalList {
+
+    ArrayList<AnimalProduct> list = new ArrayList<AnimalProduct>();
```

Comment:
I recommend specifying private or public for attributes, avoid using package-private unless it's on purpose.